build.gradle

```diff
@@ -83,14 +84,9 @@ dependencies {
     scoverage "org.scoverage:scalac-scoverage-plugin_%%:${versions.scalacScoverage}"
     scoverage "org.scoverage:scalac-scoverage-runtime_%%:${versions.scalacScoverage}"
 
-    runtime "com.h2database:h2:${versions.h2}"
-    runtime "mysql:mysql-connector-java:${versions.mysqlConnector}"
-    runtime "org.tinylog:slf4j-binding:${versions.tinylogSlf4jBinding}"
-}
-
-// exclude concrete log bindings from gatling
-configurations {
```

Comment:
exclusions for logback were showing up for every entry in the pom, extra clutter, when we really only need it for a couple entries